JSQSystemSoundPlayer/Classes/JSQSystemSoundPlayer.m

```diff
@@ -230,9 +236,11 @@ - (void)playAlertSoundWithFilename:(NSString *)filename fileExtension:(NSString
 
 - (void)playVibrateSound
 {
+#if TARGET_OS_IPHONE
```

Comment:
I thought about API compatibility here: Apple introduced new MacBook recently with «Force Touch» touchpad that can actually *vibrate*. I don't know about any public API for this at the moment, but I guess Apple may reveal it in the future.